src/core/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,136 @@
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/color-picker.css';
+
+export const DefaultColors = [
+  <Color key={1} color='#FFFFFF' name='WHITE' />,
+  <Color key={2} color='#5A6973' name='GREY' />,
+  <Color key={3} color='#00121D' name='KURETAKE BLACK MANGA' />,
+  <Color key={4} color='#002A44' name='RHAPSODY IN BLUE' />,
+  <Color key={5} color='#00426B' name='DARK IMPERIAL BLUE' />,
+  <Color key={6} color='#005A92' name='JETSKI RACE' />,
+  <Color key={7} color='#0073BA' name='FRENCH BLUE' />,
+  <Color key={8} color='#008BE1' name='BLUE COLA' />,
+  <Color key={9} color='#30B0FF' name='FANTASY CONSOLE SKY' />,
+  <Color key={10} color='#58BFFF' name='HELLO SUMMER' />,
+  <Color key={11} color='#7FCEFF' name='CHROMIS DAMSEL BLUE' />,
+  <Color key={12} color='#A6DDFF' name='DROPLET' />,
+  <Color key={13} color='#CDECFF' name='LUCID DREAMS' />,
+  <Color key={14} color='#E5F5FD' name='KODAMA WHITE' />,
+  <Color key={15} color='#010200' name='REGISTRATION BLACK' />,
+  <Color key={16} color='#122306' name='YUZU SOY' />,
+  <Color key={17} color='#23450b' name='FOREST GREEN' />,
+  <Color key={18} color='#346711' name='TATZELWURM GREEN' />,
+  <Color key={19} color='#458816' name='CHLOROPHYLL' />,
+  <Color key={20} color='#56AA1C' name='PLASTIC PINES' />,
+  <Color key={21} color='#5FBB1F' name='FIELD GREEN' />,
+  <Color key={22} color='#67CC22' name='GREEN HIGH' />,
+  <Color key={23} color='#91E458' name='LILLIPUTIAN LIME' />,
+  <Color key={24} color='#B2EC8B' name='GREEN DAY' />,
+  <Color key={25} color='#D4F4BD' name='TEA GREEN' />,
+  <Color key={26} color='#EEF6E8' name='VERDE PASTEL' />,
+  <Color key={27} color='#9BA5AF' name='SERYI GREY' />,
+  <Color key={28} color='#CF0000' name='RED EPIPHYLLUM' />,
+  <Color key={29} color='#FF6300' name='SAFETY ORANGE' />,
+  <Color key={30} color='#FFC335' name='RISE-N-SHINE' />,
+];
+
+export type ColorPickerProps = {
+  /**
+   * List of colors to show in color palette
+   * Can pass in strings of the color values for simple input or a list of Colors to customize onColorClick handler etc.
+   */
+  colors?: ReactNode[];
+  /**
+   * Style of the color picker - basic or advanced.
+   * @default 'basic'
+   */
+  type?: 'basic' | 'advanced';
+  /**
+   * Index of the selected color on the color palette.
+   */
+  activeColorIndex?: number;
+  /**
+   * Handler for clicking a color
+   * This will be applied to all colors in the palette unless specifically passed in to a specific color in the colors list
+   */
+  onColorClicked?: (index: number) => void;
+  /**
+   * Any custom nodes that will be added within the color palette
+   */
+  children?: React.ReactNode;
+};
+
+export const ColorPicker = (props: ColorPickerProps) => {
+  const {
+    colors,
+    type = 'basic',
+    activeColorIndex,
+    onColorClicked,
+    children,
+    ...rest
+  } = props;
+
+  useTheme();
+
+  // Set active color swatch
+  const [currentActiveIndex, setCurrentActiveIndex] = React.useState(
```

Comment:
YOu need to also have `React.useEffect` hook to listen to the changes of `activeColorIndex`, cause user might change it in is own way, so we need to be sure, that ColorPicker updates its state after it.

Reply:
Is this still needed after the refactoring where I removed the colors prop? Since now setting the active color is being handled in the story instead

Reply:
Yeah, now I have a question, why have you removed all of that? Dont remember we talked about removing all the logic.. Now every user will need to handle this himself.. and most of the cases, same.
Even if we moved colors to children, need to handle clicks and setting active color. You can convert children to array by using `React.Children.toArray(children)`. We have such examples with children in components like `UserIconGroup`. Or even `Fieldset` you were working with children directly.

Reply:
I removed all of it because if we try to set the active color automatically it only works if the children are ALWAYS ColorSwatches, if they are anything else it will not work. 
For example in the WithTooltip story the color swatches are wrapped in tooltips so the automatic active color will not work because the children passed in were not ColorSwatches only.
I also looked at some other examples of color picker in react and it looks like they do not have the color picker automatically setting the active color in there either, they just have the onClick handler and leave that up to the user.